.readthedocs.yaml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
how could we use this config file (mentioning requirements.txt)? can we avoid the manual installation using pip?

Reply:
No, this is for RTD.

Reply:
Oh yeah i wanted to comment on that file, sorry